Utils/layouts/index.js

```diff
@@ -20,7 +21,10 @@ import {
   const ExtendedSection = styled(Row)`
     padding-top: 30px;
     padding-bottom: 30px;
-    border-bottom: 1px solid #cbdae0;
+ 
+    ${({ bgImg }) => bgImg && `
+    background-image: url("/assets/Images/title_bg_img.png");
```

Comment:
Could we use a background image that more closely matches the design? It's also a fair bit larger than it needs to be since it is a repeated pattern.